{"stats": {"errors": 0, "failures": 1, "passes": 0, "skipped": 0, "encoding": "UTF-8", "total": 1}, "results": [{"classname": "index_test:TxTest", "name": "test_parse_inputs", "time": 0.0007987022399902344, "type": "failure", "errtype": "builtins.AssertionError", "message": "0 != 1", "tb": "Traceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n  File \"/home/5AUsK0H7q2C/tx-inputs-jsong-pb-sf-072618/test/index_test.py\", line 11, in test_parse_inputs\n    self.assertEqual(len(tx.tx_ins), 1)\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 829, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 822, in _baseAssertEqual\n    raise self.failureException(msg)\nAssertionError: 0 != 1\n"}]}